include/sound/soc-acpi.h

```diff
@@ -193,6 +194,15 @@ struct snd_soc_acpi_link_adr {
  *  is not constant since this field may be updated at run-time
  * @sof_tplg_filename: Sound Open Firmware topology file name, if enabled
  * @tplg_quirk_mask: quirks to select different topology files dynamically
+ * @get_sub_tplg_files: This is an optional callback, if specified then instead of
+ * 	the single sof_tplg_filename the callback will return the list of function topology
+ * 	files to be loaded.
+ * 	Return value: The number of the files or negative ERRNO. 0 is a valid value which means
+ * 		      the sub-topology doesn't exist.
```

Comment:
0 means that the single topology file should be used, no function topology split can be used on the machine.